app/models/user.rb

```diff
@@ -8,25 +8,21 @@ class User < ApplicationRecord
   MEMBERS = "members".freeze
   SIMPLE = "simple".freeze
 
-  ROLES = [
-    ADMIN,
-    LEAD,
-    MEMBERS,
-    SIMPLE
-  ].freeze
+  ROLES = [ADMIN, LEAD, MEMBERS, SIMPLE].freeze
+  SEX_TYPES = %w[male female no_sex].freeze
+
+  mount_uploader :avatar, AvatarUploader
 
   belongs_to :osbb, optional: true
 
   validates :first_name, presence: true, length: { maximum: 50 }
   validates :last_name, presence: true, length: { maximum: 50 }
-  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, uniqueness: true, length: { maximum: 255 }
-  validates :mobile, presence: true,
-                     numericality: true,
-                     length: { minimum: 10, maximum: 14 }
-  validates :sex, presence: true, length: { maximum: 10 }
-  validates :password, presence: true, length: { minimum: 5, maximum: 25 }
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP },
+                    uniqueness: true, length: { maximum: 255 }
+  validates :mobile, numericality: true, allow_nil: true, length: { minimum: 10, maximum: 14 }
```

Comment:
why did you remove sex validation?

Reply:
because user cannot register if sex is not set, and I also changed type of field to enum, so it does not need validation now.